modyn/tests/selector/internal/test_selector_manager.py

```diff
@@ -60,16 +54,13 @@ def __exit__(self, exc_type: type, exc_val: Exception, exc_tb: Exception):
 
 
 class MockSession:
-    def get(self, some_type, pipeline_id):  # pylint: disable=unused-argument
+    def get(self, some_type, pipeline_id) -> Optional[Pipeline]:  # pylint: disable=unused-argument
+        if pipeline_id == 0:
```

Comment:
Can we replace 0 (and the 0s in the tests below) with a global constant like `EXISTING_PIPELINE_ID`? Thank you!

Reply:
done.